README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- [x] Amazing Job so far 👏 👏 
- [x] I suggest you add emoji's to your comments so that the student understands the tone of the messege
- [x] I also strongly suggest that in order to sound polite you need to be using words like `please` `kindly` and `i suggest` and many more other worlds
- [x] Please make amends ✌️